docs/3.0.0/docs/upgrade/automation.rst

```diff
@@ -40,6 +40,7 @@ the upgrade:
 
 #. A Daml script that will be executed once by Alice and creates an
    ``UpgradeCarbonCertProposal`` contract for each owner.
+..  FIXME!(https://github.com/digital-asset/daml/issues/18650) this needs to be rewritten to avoid 'daml navigator'
 #. Navigator to accept the ``UpgradeCarbonCertProposal`` as Bob. While we
```

Comment:
this tutorial is very tied to the navigator. Should I spend any time rewriting the steps to avoid the navigator or are we planning to revamp the tutorial entirely?